docs/developer/getting-started/index.asciidoc

```diff
@@ -0,0 +1,144 @@
+[[development-getting-started]]
+== Getting started
+
+Get started building your own plugins, or contributing directly to the {kib} repo.
+
+[float]
+[[get-kibana-code]]
+=== Get the code
+
+https://help.github.com/en/github/getting-started-with-github/fork-a-repo[Fork], then https://help.github.com/en/github/getting-started-with-github/fork-a-repo#step-2-create-a-local-clone-of-your-fork[clone] the {kib-repo}[{kib} repo] and change directory into it
```

Comment:
and change the directory into it?

Reply:
I think most devs will grok "and change directory into it" better without the "the" because it references the "cd" command.